### PR TITLE
Create fundanmental flashcard functions

### PR DESCRIPTION
**Pull request by Ryoto-s**

### CODE REVIEW

**Review by ryoto-sumita-cats — COMMENTED** [review]
LGTM

**Review by ryoto-sumita-cats — APPROVED** [review]

**Review by ryoto-sumita-cats — APPROVED** [review]
LGTM

File: app/controllers/api/v1/results_controller.rb [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        process_input_enabled_flashcard(flashcard_master, word, learned_at)
+      else
+        process_input_disabled_flashcard(flashcard_master, word, learned_at)
+      end
+    end
+  end
+
+  def last_result
+    render json: {}, status: :ok
+  end
```
Comment (ryoto-cats-sub) [inline app/controllers/api/v1/results_controller.rb:32]: is it okay?

**Review by ryoto-cats-sub — COMMENTED** [review]
Check reviews

**Review by ryoto-cats-sub — COMMENTED** [review]
You are almost there!

**Review by ryoto-cats-sub — APPROVED** [review]
yay you are all set

**Pull request merged**